### PR TITLE
Docs tweak for profiler

### PR DESCRIPTION
**Pull request by nodebotanist**

### CODE REVIEW

File: PROFILER.md [hunk 1]
```diff
@@ -9,21 +9,19 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environwental variable `IOPIPE_PROFILER_ENABLE` to `true`
```
Comment (pselle) [inline PROFILER.md]: This is serverless framework specific ... the language should be general to "set an environment variable" and then this section for serverless framework would be a bonus

File: PROFILER.md [hunk 2]
```diff
@@ -9,21 +9,19 @@ which inspects the state of all threads to determine how long they have been
...
 
-The profiler may be customed by setting the specified properties and/or
+The profiler may be customized by setting the specified properties and/or
 environment variables:
 
  * The duration of time between each sample, in microseconds:
    * The default is 1,000 microseconds (1ms).
-   * `com.iopipe.plugin.profiler.samplerate`
-   * `IOPIPE_PROFILER_SAMPLERATE`
+   * `IOPIPE_PROFILER_SAMPLERATE` in the environmental varibles of the `serverless.yml` file
```
Comment (pselle) [inline PROFILER.md]: Same here, serverless.yml is serverless framework specific

File: PROFILER.md [hunk 3]
```diff
@@ -9,21 +9,19 @@ which inspects the state of all threads to determine how long they have been
...
 
-When the profiler is enabled, they may be downloaded from the individual
-invocation information.
+When the profiler is enabled, .zip files may be downloaded from the individual
```
Comment (pselle) [inline PROFILER.md]: ".zip files containing the profiling data"

File: PROFILER.md (hunk 1 above)
Comment (nodebotanist) [reply PROFILER.md]: Fixed in new commit

File: PROFILER.md (hunk 2 above)
Comment (nodebotanist) [reply PROFILER.md]: Also fixed in new commit

File: PROFILER.md (hunk 3 above)
Comment (nodebotanist) [reply PROFILER.md]: Fixed in new commit

**Comment by nodebotanist** [comment]
@pselle squashed commits, ready for re-review

File: PROFILER.md [hunk 4]
```diff
@@ -9,21 +9,24 @@ which inspects the state of all threads to determine how long they have been
...
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+### In General:
 
-The profiler may be customed by setting the specified properties and/or
+Set the environment variable `com.iopipe.plugin.profiler` to `true`
```
Comment (pselle) [inline PROFILER.md]: This isn't the environment variable, but a system property. There should also be a newline below this in markdown.

File: PROFILER.md [hunk 5]
```diff
@@ -9,21 +9,24 @@ which inspects the state of all threads to determine how long they have been
...
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environwental variable `IOPIPE_PROFILER_ENABLE` to `true`
+
+The profiler may be customized by setting the specified properties and/or
```
Comment (pselle) [inline PROFILER.md]: In this instruction, you mention properties, but you dropped it from bullets the docs below. One or the other (either mention properties and keep it in the docs, or drop the mention of properties)

File: PROFILER.md [hunk 6]
```diff
@@ -9,21 +9,24 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+### In General:
```
Comment (pselle) [inline PROFILER.md]: The "in general" is redundant ... you're telling people they can set the environment variable, but then the following header is for serverless framework specifically

File: PROFILER.md [hunk 7]
```diff
@@ -9,21 +9,23 @@ which inspects the state of all threads to determine how long they have been
...
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+Set the system property `com.iopipe.plugin.profiler` to `true`
 
-The profiler may be customed by setting the specified properties and/or
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environwental variable `IOPIPE_PROFILER_ENABLE` to `true`
```
Comment (ewindisch) [inline PROFILER.md]: "environwental" typo

Comment (nodebotanist) [reply PROFILER.md]: Ah thanks!

**Comment by nodebotanist** [comment]
I added a header to clear up the wording issue around properties, and fixed the typo.

File: PROFILER.md [hunk 8]
```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+Set the system property `com.iopipe.plugin.profiler` to `true`
```
Comment (pselle) [inline PROFILER.md]: This should be set the environment variable (or mention both), you can't set system properties on Lambda, the system property _is_ available in the system should someone be running some sort of alternate setup

File: PROFILER.md [hunk 9]
```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
...
+## Customizing Profiling
+
+The profiler may be customized by setting the specified properties and/or
 environment variables:
 
  * The duration of time between each sample, in microseconds:
    * The default is 1,000 microseconds (1ms).
-   * `com.iopipe.plugin.profiler.samplerate`
-   * `IOPIPE_PROFILER_SAMPLERATE`
+   * `IOPIPE_PROFILER_SAMPLERATE` in the environmental varibles of the `serverless.yml` file, or the environment variable `com.iopipe.plugin.profiler.samplerate`.
```
Comment (pselle) [inline PROFILER.md]: varibles => variables and `com.iopipe....` is not an environment variable, you can drop the mention if you don't want to cover system properties.

File: PROFILER.md [hunk 10]
```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
...
+Set the environment variable IOPIPE_PROFILER_ENABLE to `true`
 
-The profiler may be customed by setting the specified properties and/or
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environmental variable `IOPIPE_PROFILER_ENABLE` to `true`
+
+## Customizing Profiling
+
+The profiler may be customized by setting the specified properties and/or
```
Comment (pselle) [inline PROFILER.md]: This no longer references "specified properties"

File: PROFILER.md [hunk 11]
```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
...
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+Set the environment variable IOPIPE_PROFILER_ENABLE to `true`
 
-The profiler may be customed by setting the specified properties and/or
+### For Serverless Framework:
+
+In the `serverless.yml` file, under `functions:`, `(your_function_name_here):`, `environment:`, set the environmental variable `IOPIPE_PROFILER_ENABLE` to `true`
```
Comment (pselle) [inline PROFILER.md]: environmental variable => environment variable

File: PROFILER.md [hunk 12]
```diff
@@ -9,21 +9,25 @@ which inspects the state of all threads to determine how long they have been
...
+## Customizing Profiling
+
+The profiler may be customized by setting the specified properties and/or
 environment variables:
 
  * The duration of time between each sample, in microseconds:
    * The default is 1,000 microseconds (1ms).
-   * `com.iopipe.plugin.profiler.samplerate`
-   * `IOPIPE_PROFILER_SAMPLERATE`
+   * `IOPIPE_PROFILER_SAMPLERATE` as an environmental variable
```
Comment (pselle) [inline PROFILER.md]: environmental variable => environment variable

**Pull request merged**